op-service/txmgr/send_state.go

```diff
@@ -27,12 +27,12 @@ type SendState struct {
 	now      func() time.Time
 
 	// Config
-	nonceTooLowCount    uint64
-	txInMempoolDeadline time.Time // deadline to abort at if no transactions are in the mempool
+	safeAbortNonceTooLowCount uint64
+	txInMempoolDeadline       time.Time // deadline to abort at if no transactions are in the mempool
 
 	// Counts of the different types of errors
-	successFullPublishCount   uint64 // nil error => tx made it to the mempool
-	safeAbortNonceTooLowCount uint64 // nonce too low error
+	successFullPublishCount uint64 // nil error => tx made it to the mempool
+	nonceTooLowCount        uint64 // nonce too low error
```

Comment:
just swapped the position of these variables (the "config" was under the "count" and vice-versa)